plugins/jobs/src/js/JobsRunNow.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-unused-vars */
+import React from "react";
+/* eslint-enable no-unused-vars */
+
+import { graphqlObservable } from "data-service";
+import gql from "graphql-tag";
+
+import { Subject } from "rxjs/Subject";
+import { defaultSchema } from "./JobsMenuClient";
+
+const runNowJobMutation = gql`
+  mutation {
+    runJob(jobId: $jobId) {
```

Comment:
How does GraphQL know what $jobId is? You only pass a variable to the graphqlObservable, right?

Reply:
Changed to id: $id. I'm not 100% how does this work, but if I put id: String or something else it doesn't map correctly with the defaultSchema. Probably @Fabs can help us here.